packet_forwarder/src/lora_pcap.c

```diff
@@ -0,0 +1,355 @@
+/* fix an issue between POSIX and C99 */
+#if __STDC_VERSION__ >= 199901L
+    #define _XOPEN_SOURCE 600
+#else
+    #define _XOPEN_SOURCE 500
+#endif
+
+#include <arpa/inet.h>
+#include <sys/types.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+
+#include "loragw_hal.h"
+#include "lora_pcap.h"
+#include "cursor/cursor.h"
+
+#define LINKTYPE_LORATAP 270
+#define LW_SYNC 0x34
+
+/*
+ * The PCAP file header. Using clever runtime checking of the
+ * magic number field, this structure can be written to disk
```

Comment:
I see a write method for this struct, but no 'read'. Is that done elsewhere? Sorry if dump question, I don't know a whole lot about pcap